package.json

```diff
@@ -16,7 +16,8 @@
   },
   "dependencies": {
     "@tf2-pickup/configs": "2.1.0",
-    "@tf2-pickup/tf2-configs": "2.0.1",
+    "@tf2-pickup/tf2-configs": "^2.0.1",
+    "await-sleep": "^0.0.1",
```

Comment:
This is actually not needed because we already have a sleep package called promise-sleep

Reply:
Got it